src/mjolnir/adminbuilder.cc

```diff
@@ -75,83 +49,47 @@ int polygondata_comparearea(const void* vp1, const void* vp2) {
   return 1;
 }
 
-std::vector<std::string> GetWkts(std::unique_ptr<Geometry>& mline) {
-  std::vector<std::string> wkts;
-
-#if 3 == GEOS_VERSION_MAJOR && 6 <= GEOS_VERSION_MINOR
-  auto gf = GeometryFactory::create();
-#else
-  std::unique_ptr<GeometryFactory> gf(new GeometryFactory());
-#endif
-
-  LineMerger merger;
-  merger.add(mline.get());
-#if GEOS_VERSION_MAJOR == 3 && GEOS_VERSION_MINOR >= 9
-  std::vector<std::unique_ptr<LineString>> merged(merger.getMergedLineStrings());
-#else
-  std::unique_ptr<std::vector<LineString*>> merged(merger.getMergedLineStrings());
-#endif
-  WKTWriter writer;
```

Comment:
I didn't fully understand why there was so much geometry massaging going on when the end geometry was actually pretty simple. probably some geos niceties.. that could now be simplified a lot. basically we already give this function polygons, do the processing based on them and then later build up the final `MultiPolygon` which is returned as WKT. no intermediary geometry massaging